src/test_graph.py

```diff
@@ -2,229 +2,222 @@
 
 #test node constructor
 
-def test_node_init_value(gnode1):
-    assert gnode1.value == 1
-
-
-def test_node_init_edges(gnode1):
-    assert gnode1.edges == set()
 
 def test_graph_init_set_of_nodes(graph_empty):
     assert graph_empty.set_of_nodes == set()
 
-def test_graph_is_node_true(graph_empty, gnode1):
-    assert graph_empty.is_node(gnode1)
-
-def test_graph_is_node_error(graph_empty):
-    with pytest.raises(TypeError):
-        graph_empty.is_node('not_a_node')
+def test_graph_add_node(graph_empty):
+    graph_empty.add_node('gnode1')
+    assert 'gnode1' in graph_empty.set_of_nodes
 
-def test_graph_add_node(graph_empty, gnode1):
-    graph_empty.add_node(gnode1)
-    assert gnode1 in graph_empty.set_of_nodes
+def test_graph_has_node_true(graph_empty):
+    graph_empty.add_node('gnode1')
+    assert graph_empty.has_node('gnode1') is True
```

Comment:
Also, your linter should be telling you about this: keep a consistent 2 lines between each function.